Rakefile

```diff
@@ -2,6 +2,11 @@ require 'rubocop/rake_task'
 require 'rake/testtask'
 require 'rspec/core/rake_task'
 
+def valid_gem?(input_gem)
+  valid_gems = %w(core grpc nxapi)
```

Comment:
Why do you only check for 'core' below and not also the other two?

And why if GEM=core builds nxapi - what does GEM=nxapi do?

What about node utils?